app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx

```diff
@@ -408,9 +409,14 @@ export class BtsPropertiesForIssueForm extends Component {
   };
 
   render() {
-    const { intl, disabled } = this.props;
+    const { intl, disabled, integrationId, pluginName, projectName, projectInfo } = this.props;
     const { loading } = this.state;
     const preparedFields = this.prepareFieldsToRender();
+    const integrationInfo = {
+      integrationId,
+      projectName: projectName || projectInfo.projectName,
```

Comment:
Do we really need to use projectInfo.projectName?
These values could be not equal.

Reply:
I suppose the answer is 'yes' until the backend add a support of global integration based command to avoid using project name.
This hack is used for integrations on admin level.